extensions/arc/runtime/pom.xml

```diff
@@ -27,6 +27,17 @@
       <groupId>org.eclipse.microprofile.context-propagation</groupId>
       <artifactId>microprofile-context-propagation-api</artifactId>
     </dependency>
+    <dependency>
+      <groupId>xalan</groupId>
```

Comment:
Why ARC needs to depend on Xalan?

Reply:
Hi @gastaldi ,
If I put it on the other places the tika-integration-tests are failed.